### PR TITLE
Feat 494 event dates

### PR DESCRIPTION
**Pull request by opyate**

### CODE REVIEW

File: backend/app/db/models.py [hunk 1]
```diff
@@ -343,7 +343,7 @@ class Event(Base):  # noqa: D101
     )
     name = sa.Column(sa.Text, nullable=False)
     description = sa.Column(sa.Text, nullable=False)
-    created_ts = sa.Column(sa.DateTime(timezone=True), server_default=func.now())
```
Comment (joel-wright) [inline backend/app/db/models.py:346]: I thought we also decided to add an explicit year (or even date) field to the `Document` model too?

**Review by kdutia — APPROVED** [review]
lgtm!

**Review by joel-wright — APPROVED** [review]
LGTM, just the one question about the explicit year/date we wanted to add to the `Document` table. That can be done in a follow-up if we still decide we need it after a quick chat.

File: backend/app/db/models.py (hunk 1 above)
Comment (opyate) [reply backend/app/db/models.py:346]: Yes, after merging your currently open PR?

Comment (opyate) [reply backend/app/db/models.py:346]: Yes, after merging your currently open PR?

**Pull request merged**